targettests/Remote-Sim/no_inline.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
```

Comment:
Nit since this is a new file.
```suggestion
 * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
```